hadoop-gremlin/src/main/java/org/apache/tinkerpop/gremlin/hadoop/structure/io/script/ScriptOutputFormat.java

```diff
@@ -29,6 +29,12 @@
 import java.io.IOException;
 
 /**
+ * {@link org.apache.tinkerpop.gremlin.hadoop.structure.io.script.ScriptInputFormat}
+ * and ScriptOutputFormat take an arbitrary script and use that script to either
+ * read or write Vertex objects, respectively. This can be considered the most
+ * general InputFormat/OutputFormat possible in that Hadoop-Gremlin uses the user
+ * provided script for all reading/writing.
+ * @see <a href="http://tinkerpop.apache.org/docs/3.2.0-incubating/reference/#script-io-format">Script I/O Format Reference Documentation</a>
```

Comment:
good point - i didn't think of that. right now we don't have automation for that.